### PR TITLE
Update VisualStyleElement.xml

### PR DESCRIPTION
**Pull request by RussKie**
## Summary

Point to the correct documentation

### CODE REVIEW

File: xml/System.Windows.Forms.VisualStyles/VisualStyleElement.xml [hunk 1]
```diff
@@ -81,7 +81,7 @@
 ## Remarks  
  A class name refers to a group of related visual style elements that are typically drawn together. A class name can correspond to a control, such as "SCROLLBAR" and "TRACKBAR", but a class name can also refer to a group of elements that do not correspond to a particular control, such as "EXPLORERBAR" and "STARTPANEL".  
   
- For more information on visual style classes, parts, and states, see "Parts and States" in the Windows Shell documentation at [https://msdn.microsoft.com/library](https://msdn.microsoft.com/library).  
+ For more information on visual style classes, parts, and states, see "Parts and States" in the Desktop App User Interface documentation at [https://docs.microsoft.com/windows/win32/controls/parts-and-states](https://docs.microsoft.com/windows/win32/controls/parts-and-states).
```
Comment (gewarren) [inline xml/System.Windows.Forms.VisualStyles/VisualStyleElement.xml:84]:
```suggestion
 For more information on visual style classes, parts, and states, see [Parts and States](https://docs.microsoft.com/windows/win32/controls/parts-and-states).
```

File: xml/System.Windows.Forms.VisualStyles/VisualStyleElement.xml [hunk 2]
```diff
@@ -151,7 +151,7 @@
 ## Remarks  
  A part is a discrete member of a class of visual style elements. For example, visual style elements that represent a check box, radio button, and push button are all considered parts of the "BUTTON" class.  
   
- Part values are integers that range from 1 to the number of parts defined for a particular class. For more information on visual style classes, parts, and states, see "Parts and States" in the Windows Shell documentation at [https://msdn.microsoft.com/library](https://msdn.microsoft.com/library).  
+ Part values are integers that range from 1 to the number of parts defined for a particular class. For more information on visual style classes, parts, and states, see "Parts and States" in the Desktop App User Interface documentation at [https://docs.microsoft.com/windows/win32/controls/parts-and-states](https://docs.microsoft.com/windows/win32/controls/parts-and-states).
```
Comment (gewarren) [inline xml/System.Windows.Forms.VisualStyles/VisualStyleElement.xml:154]:
```suggestion
 Part values are integers that range from 1 to the number of parts defined for a particular class. For more information on visual style classes, parts, and states, see [Parts and States](https://docs.microsoft.com/windows/win32/controls/parts-and-states).
```

File: xml/System.Windows.Forms.VisualStyles/VisualStyleElement.xml [hunk 3]
```diff
@@ -190,7 +190,7 @@
 ## Remarks  
  A visual style part can have one or more visual states. For example, a visual style element that represents a radio button can be in the disabled, checked, or unchecked state.  
   
- State values are integers that range from 1 to the number of states defined for a particular part. For more information on visual style classes, parts, and states, see "Parts and States" in the Windows Shell documentation located at [https://msdn.microsoft.com/library](https://msdn.microsoft.com/library).  
+ State values are integers that range from 1 to the number of states defined for a particular part. For more information on visual style classes, parts, and states, see "Parts and States" in the Desktop App User Interface documentation at [https://docs.microsoft.com/windows/win32/controls/parts-and-states](https://docs.microsoft.com/windows/win32/controls/parts-and-states).
```
Comment (gewarren) [inline xml/System.Windows.Forms.VisualStyles/VisualStyleElement.xml:193]:
```suggestion
 State values are integers that range from 1 to the number of states defined for a particular part. For more information on visual style classes, parts, and states, see [Parts and States](https://docs.microsoft.com/windows/win32/controls/parts-and-states).
```

**Comment by BillWagner** [comment]
It looks like there are a couple other changes to make, and then this is ready.

**Pull request merged**